index.html

```diff
@@ -112,13 +111,15 @@ <h3>How is it different from Crouton?</h3>
         <p>
           The idea is that you may be on a weak internet connection and cannot download too much data, but you don't have Crouton and need just a few small packages. Also, you may be on a good internet connection and need just a few small packages. Also, why not use Chrome OS as the operating system?
         </p>
-
+<h3>Is GUI supported?</h3>
+<p>
+  In fact, Chromebrew supports GUI, but i686 is not supported for GUI apps.
```

Comment:
Please change to `Chromebrew supports most GUI apps with help from the sommelier daemon.  We don't support i686 architecture, however, since Google has dropped support.`